app/Http/Controllers/EmployeeController.php

```diff
@@ -197,4 +197,62 @@ public function getDirectoryEmployees()
         $data = $fractal->createData($collection)->toArray();
         return $this->respond($data);
     }
+
+    /* Additions by John begin */
```

Comment:
@John-Ip  - in the future you don't need to add comments like these, since Git automatically keeps track of who made what changes when